plugins/out_stackdriver/stackdriver.c

```diff
@@ -446,6 +447,41 @@ static int get_severity_level(severity_t * s, const msgpack_object * o,
     return -1;
 }
 
+static int pack_json_payload(bool operation_extracted, int operation_extra_size, 
+                            msgpack_packer* mp_pck, msgpack_object *obj)
+{
+    /* Specified fields include operation, sourceLocation ... */
+    int to_remove = 0;
+    if(operation_extracted && operation_extra_size == 0) {
+        to_remove += 1;
+    }
+
+    int ret = msgpack_pack_map(mp_pck, obj->via.map.size - to_remove);
+    if(ret < 0) {
+        return ret;
+    }
+    else {
```

Comment:
cosmetic change: get rid of the ```else``` statement, sense all lines below will run if ret >= 0 (that helps to reduce line length too)

Reply:
Done.